_rules/text-contrast-afw4f7.md

```diff
@@ -31,7 +31,7 @@ acknowledgments:
 This rule applies to any [visible][] character in a [text node][] that is a [child][] in the [flat tree][] of an HTML element, except if the [text node][] has an [ancestor][] in the [flat tree][] for which one of the following is true:
 
 - **widget**: the ancestor has a [semantic role][] that inherits from `widget`; or
-- **disabled label**: the ancestor is used in the [accessible name][] of a `widget` that is [disabled][]; or
+- **disabled label**: the ancestor is used in the [accessible name][] of an element with a role inheriting from `widget` that is [disabled][]; or
```

Comment:
You know it already. Wilco's going to ask you to do this in a different PR ;-)

Reply:
🙈 I though I did that in #1532